### PR TITLE
Fix prompt template docs for ~/.config/zed/prompt_overrides

### PR DESCRIPTION
**Pull request by g0t4**
The docs had an old path 

Also pointed at the template files instead of duplicating the contents in the docs which will inevitably get out of date again.

By the way, [step_resolution.hbs](https://github.com/zed-industries/zed/tree/main/assets/prompts/step_resolution.hbs) is not currently in the zed repo, that would need to be added if it's still used.

Release Notes:

- Updated zed.dev/docs to link to assistant prompt templates in the zed repo

### CODE REVIEW

**Comment by g0t4** [comment]
@cla-bot check

**Comment by g0t4** [comment]
Here is the relevant code that uses this path:

https://github.com/zed-industries/zed/blob/07821083dff23fa7593b5d35c2fc119445417668/crates/paths/src/paths.rs#L235-L244

File: docs/src/assistant/prompting.md [hunk 1]
```diff
@@ -146,78 +146,19 @@ At this point it is unknown if we will expand templates further to be user-creat
...
-   {{user_prompt}}
-
-   Provide a clear and concise response to the user's query, considering the
-   given system information and latest terminal output if relevant.
-   ```
-
-3. `suggest_edits.hbs`: Used for generating the model instructions for the XML Suggest Edits should return.
-
-4. `step_resolution.hbs`: Used for generating the step resolution prompt.
+1. [`content_prompt.hbs`](../../../assets/prompts/content_prompt.hbs): Used for generating content in the editor.
```
Comment (notpeter) [inline docs/src/assistant/prompting.md:153]: Pretty sure these links need to be like:
https://github.com/zed-industries/zed/blob/main/assets/prompts/content_prompt.hbs
to work on zed.dev/docs/

Comment (g0t4) [reply docs/src/assistant/prompting.md:153]: thanks, eagle eye, I'll push an update

Comment (g0t4) [reply docs/src/assistant/prompting.md]: fixed

**Comment by g0t4** [comment]
Let me know if you prefer a rebase, I can do that before merging

**Comment by notpeter** [comment]
Thank you for this. 
I went ahead and removed `step_resolution.hbs` as that is no longer used.

**Pull request merged**